README.md

```diff
@@ -1,60 +1,56 @@
 endless hordes of enemies!
 
-The game is developed by the "Prosjekt: Spill" group of the Hackerspace NTNU student organization over the span of 2 years.
+The game was developed by the "Prosjekt: Spill" group of the Hackerspace NTNU student organization over the span of 2 years.
 The project is open-source and free for all who are interested.
 For other projects and information, take a look at [Hackerspace's website](https://www.hackerspace-ntnu.no/).
 
-In 2022, Temple Trashers was nominated for the "Most Innovative Game Design" award at the Norwegian Game Awards! 
+In 2022, Temple Trashers was nominated for the "Most Innovative Game Design" award at the [Norwegian Game Awards](https://www.gameawards.no/)!
 
-## Relevant pages
-[Steam](https://store.steampowered.com/app/1984140)
 
-[Itch.io](https://hackerspace.itch.io/temple-trashers)
+## Relevant pages
+* [Steam](https://store.steampowered.com/app/1984140)
+* [Itch.io](https://hackerspace.itch.io/temple-trashers)
 
 
 ## Development, feedback and issues
-Although active official development on this project is concluded as per the Hackerspace NTNU guidelines, the core team members retains access to manage the Github, Steam and Itch.io pages, or even continue developing at their own volition.  
+Although active official development on this project has concluded as per the Hackerspace NTNU guidelines,
+the core team members retain access for managing the Github, Steam and Itch.io pages, and are free to continue developing - at their own volition.
 
-If you have found a bug, have feedback or a suggestion, please create an issue in this repository.
+If you have found a bug, or want to provide feedback or come with a suggestion,
+please [create an issue](https://github.com/hackerspace-ntnu/Temple-Trashers/issues/new/choose).
 
-If you want to contribute to this game's development yourself, feel free to fork this project and create a pull request!
+If you want to contribute to this game's development yourself,
+feel free to fork this project and [create a pull request](https://github.com/hackerspace-ntnu/Temple-Trashers/compare)!
 
-The team uses Trello for our agile development process, which is available for public view [here.](https://trello.com/b/Ht2dAEOh/prosjekt-spill-utvikling)
+The team used Trello for our agile development process, and [our Trello board](https://trello.com/b/Ht2dAEOh/prosjekt-spill-utvikling) is publicly open.
 
 
-## Team members (At time of release)
+## Team members (at time of release)
 
 ### Developers
-[ddable](https://github.com/ddabble)
-
-[Fueredoriku](https://github.com/Fueredoriku)
-
-[jesper2k](https://github.com/jesper2k)
-
-[SondreHus](https://github.com/SondreHus)
-
-[TArchetype](https://github.com/TArchetype)
+* [ddabble](https://github.com/ddabble)
+* [Fueredoriku](https://github.com/Fueredoriku)
+* [jesper2k](https://github.com/jesper2k)
+* [SondreHus](https://github.com/SondreHus)
+* [TArchetype](https://github.com/TArchetype)
```

Comment:
Legge til våre fulle navn her også?

Reply:
Vi kan godt det for min del, men da må vi høre med folk om de er komfortable med det først :)

Reply:
@ddabble @TArchetype @SondreHus er dere komfortable med at vi har navnene vår her?

Reply:
For min del vil jeg gjerne ha enten bare fornavnet mitt eller brukernavnet mitt 🙂 Kanskje spørre på slack, slik at de andre vi ikke har tagga her også kan svare 😊